e2e-tests/test/main.test.ts

```diff
@@ -62,11 +62,18 @@ describe("Greeter Smart Contract", function () {
 
     // Validate log is created
     expect(receipt.logs.length).to.greaterThanOrEqual(1);
-    const setGreetingLog = receipt.logs[0];
-    expect(setGreetingLog.address).to.equal(greeter.address);
+    let setGreetingLog = null;
+    for (let i = 0; setGreetingLog == null && i < receipt.logs.length; ++i) {
+      if (receipt.logs[i].address == greeter.address) {
+        setGreetingLog = receipt.logs[i];
```

Comment:
I see that this contract emits this event:
`emit LogString(string.concat("Greeting is being updated to ", _greeting));`
Can we check that the log is exactly `Greeting is being updated to Luke Skywalker`
would be nice